### ISSUE TITLE
🚀 Feature: On share page, next to download button - copy direct link button

### ISSUE THREAD

**Issue by infestdead** [stonith404/pingvin-share#67]
### 🔖 Feature description

On share page, next to the download button/icon - it would be really nice to have a - copy direct link to file button/icon.
You should be able to wget/curl the file in question for anywhere.

### 🎤 Pitch

It would be very useful to get a direct link to the file you want to download. 
That's useful when you open the link on a browser, but want to download the file on a remote computer via curl/wget etc.

**Comment by stonith404** [comment]
See #21 why it isn't possible. If you have an idea how to solve this let me know :)

**Comment by infestdead** [comment]
I already use this is a way, when I click download and it starts downloading I stop it and from firebug/dev tools I get the full link that I can then curl/wget on my server. 
Also if you do a curl/wget you would normally always want a single file anyway, so not really an issue I think. As for the view count - each download can be counted as a view, also a warning can be displayed that any person with the direct link can download the file, no sign-in needed - kind of like google docs does it.

**Comment by stonith404** [comment]
@infestdead The problem is that the link is only valid for 15minutes. Would that be enough for you?

**Comment by infestdead** [comment]
That would work great for me, and I really think it's a useful addition. It could be a toggle in the options (Enable direct links to files) or something along those lines.

**Comment by sebasdt** [comment]
I want to extent a little bit on this. 15 minutes should be enough

**Comment by stonith404** [comment]
I thought about an option like "allow direct downloads" that can be enabled or not. If enabled you can't use any security features. What do you guys think?

**Comment by infestdead** [comment]
What do you mean by "any security features"? If it's the Password protection and Max Views - I think it's a good compromise.

**Comment by stonith404** [comment]
Yeah I think too, the security features that would be disabled are password and max views

**Comment by stonith404** [comment]
Added in `v0.9.0` :) A direct file link can only be used if the share hasn't a password.

**Comment by infestdead** [comment]
Tested, works great, thanks a lot :+1:
